### PR TITLE
Update caching_with_rails.md

### PR DESCRIPTION
**Pull request by enowmbi**
[ci skip] Added line describing how to toggle caching using rails dev:cache in development mode

### CODE REVIEW

**Comment by enowmbi** [comment]
First time contributing to rails documentation. why are some checks not successful ? Does it mean the commit was not merged ?

**Comment by p8** [comment]
@enowmbi I think you have to put `[ci skip]` in the first line of the pull request to skip the buildkite check.

File: guides/source/caching_with_rails.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ only have an effect on the caching provided by the Action Controller component.
 For instance, it will not impact low-level caching, that we address
 [below](#low-level-caching).
 
+In Rails 5 and above, instead of editing the config/environment/development.rb file, you can toggle caching by running rails dev:cache command; running this command enables caching if not enabled, and  disables caching if it is enabled.
+
```
Comment (p8) [inline guides/source/caching_with_rails.md]: At the bottom of the page there already is an explanation on how to use `rails dev:cache`. Maybe move that up instead?

Comment (enowmbi) [reply guides/source/caching_with_rails.md]: Thanks for your response.

**Pull request closed without merging**